core/arch/arm/plat-stm32mp1/scmi_server.c

```diff
@@ -205,12 +207,9 @@ struct stm32_scmi_voltd scmi_voltage_domain[] = {
 	VOLTD_CELL(VOLTD_SCMI_REG11, VOLTD_PWR, PWR_REG11_NAME_ID, "reg11"),
 	VOLTD_CELL(VOLTD_SCMI_REG18, VOLTD_PWR, PWR_REG18_NAME_ID, "reg18"),
 	VOLTD_CELL(VOLTD_SCMI_USB33, VOLTD_PWR, PWR_USB33_NAME_ID, "usb33"),
-	VOLTD_CELL(VOLTD_SCMI_SDMMC1_IO, VOLTD_PWR, PWR_SDMMC1_IO_NAME_ID,
-		   "sdmmc1"),
-	VOLTD_CELL(VOLTD_SCMI_SDMMC2_IO, VOLTD_PWR, PWR_SDMMC2_IO_NAME_ID,
-		   "sdmmc2"),
-	VOLTD_CELL(VOLTD_SCMI_VREFBUF, VOLTD_PWR, PWR_VREFBUF_NAME_ID,
-		   "vrefbuf"),
+	VOLTD_CELL(VOLTD_SCMI_SDMMC1_IO, VOLTD_STUB, 0, "sdmmc1"),
+	VOLTD_CELL(VOLTD_SCMI_SDMMC2_IO, VOLTD_STUB, 0, "sdmmc2"),
+	VOLTD_CELL(VOLTD_SCMI_VREFBUF, VOLTD_STUB, 0, "vrefbuf"),
```

Comment:
use different index to check on index ? 0 = SDMMC1/2  and 0 = VREFBUF

VOLTD_CELL(VOLTD_SCMI_VREFBUF, VOLTD_STUB, 1, "vrefbuf"),

Reply:
OK